dev-guide/index.md

```diff
@@ -0,0 +1,331 @@
+# How to Game Jolt: Developer's Guide
+
+## Create a Developer Account
+
+The first step is creating a developer account. If you're reading this, there's a good chance you already have a developer account on Game Jolt. If not, read on.
+
+If you don't have a Game Jolt account at all, go ahead and [sign up](http://gamejolt.com/auth/sign_up/) for one.   
+
+If you already have a gamer account on Game Jolt, it's a cinch to become a developer. Just hover your mouse over "Dashboard" (on the navigation bar at the top of the page) until a new menu appears, then click/tap "Become a Developer".
+
+### Your Avatar
```

Comment:
we should move this out of the "dev" guide and into a general guide since avatars are used by both devs and gamers.